dcos/DCOS.md

```diff
@@ -106,19 +97,18 @@ dcos marathon app add dcos/marathon/websvc.json
 
 ## Confirm it all works
 
-### Initialize namerd dtabs
+### View namerd dtabs
 
 ```bash
-curl -vX POST -d @dcos/namerd.dtab --header "Content-Type: application/dtab" $PUBLIC_URL:4180/api/1/dtabs/default
-open $PUBLIC_URL:4180/api/1/dtabs/default
+curl $PUBLIC_URL:4180/api/1/dtabs/default
 ```
 
 ### Update namerd dtabs
 
 ```bash
 ETAG=`curl -i $PUBLIC_URL:4180/api/1/dtabs/default|grep -Fi ETag: | awk {'print $2'} | sed -e 's/[[:cntrl:]]//'`
 curl -vX PUT -d @dcos/namerd.dtab --header "Content-Type: application/dtab" --header "If-Match: $ETAG" $PUBLIC_URL:4180/api/1/dtabs/default
-open $PUBLIC_URL:4180/api/1/dtabs/default
+curl $PUBLIC_URL:4180/api/1/dtabs/default
```

Comment:
Can defer this, but it's probably much clearer to just use `namerctl` as illustrated in https://github.com/BuoyantIO/linkerd-examples/pull/3/files